### PR TITLE
Issue#32

### PR DESCRIPTION
**Pull request by mugich**
created signup.js
added signup form jquery
add api/signup router

### CODE REVIEW

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -15,8 +15,9 @@
     
     <!-- CSS Stylesheet -->
     <link rel="stylesheet" href="/assets/css/style.css" type="text/css" />
-
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
```
Comment (ncastaldi) [inline views/layouts/main.handlebars:18]: Isn't there a jQuery reference a few lines above this?

**Review by ncastaldi — CHANGES_REQUESTED** [review]
Could you look at main.handlebars? I think there's already a jQuery call a few lines above 18.

File: views/layouts/main.handlebars (hunk 1 above)
Comment (mugich) [reply views/layouts/main.handlebars:18]: yeah got it i'll remove it

**Comment by mugich** [comment]
can you do change request again? I cannot remove it

**Pull request closed without merging**